active_learning/learn.py

```diff
@@ -0,0 +1,136 @@
+import numpy
+
+from scipy.sparse import vstack
+
+from sklearn.model_selection import StratifiedKFold, train_test_split
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.linear_model import SGDClassifier
+from sklearn.metrics import confusion_matrix, precision_recall_fscore_support, accuracy_score
+
+from libact.base.dataset import Dataset
+from libact.query_strategies import *
+from libact.models import *
+
+from active_learning.models import Article, Label, DataSet
+from active_learning.libact_models import *
+
+
+def train_active_learning(num_queries, training_dataset, oracle_training_dataset):
+    active_learning_strategy = 1
+    query_strategy, strategy = get_active_learning_strategy(active_learning_strategy, training_dataset)
+    model = SGDClassifier(loss='hinge', penalty='l2', alpha=1e-2, n_iter=1000, random_state=None, learning_rate='optimal', class_weight='balanced')
+    # train on labeled instances
+    model.fit(*(training_dataset.format_sklearn()))
+    # active learning
```

Comment:
Try to reuse the code from learning.py. Create a separate method with condition statements that determine whether to produce intermediate results